pkg/lib/config/feature_oauth.go

```diff
@@ -20,14 +20,16 @@ var _ = FeatureConfigSchema.Add("OAuthClientFeatureConfig", `
 	"additionalProperties": false,
 	"properties": {
 		"maximum": { "type": "integer" },
-		"custom_ui_enabled": { "type": "boolean" }
+		"custom_ui_enabled": { "type": "boolean" },
+		"app2app_enabled": { "type": "boolean" }
```

Comment:
The server should also respect this config. If app2app_enabled is false, then app2app request should fail with App2AppDisabled error.